docs/man/docker-run.1.md

```diff
@@ -336,6 +336,10 @@ mount is not running.
 default, the volumes are mounted in the same mode (read write or read only) as
 the reference container.
 
+   If the location of the volume from the specified container overlaps with
```

Comment:
Source/target or source/destination are good adjectives to use -- this helps keep the two locations distinct and reinforce the relationship in the reader's mind. You could apply source/target to the container or to the volume just be consistent. In this instance I used container as it made sense
-----

**--volumes-from**=[]
   Mount volumes from the specified container(s)

Mounts already mounted volumes from a source container onto one or more additional, target containers. You must supply the source container's container-id. To share a volume, use the **--volumes-from** option when running
the target containers. You can share volumes even if the source container is not
running.

 By default, Docker mounts the volumes the same mode (read-write or read-only)
 as it the mount on the source container. Optionally, you change this  by
 suffixing the container-id with either the `:ro` or `:rw ` keyword.

If the location of the volume from the source container overlaps with
data residing on a target container, then the volume hide
that data on the target.